src/polyswarm_api/__init__.py

```diff
@@ -372,14 +373,20 @@ def _fix_result(self, result):
         with open(to_scan, 'rb') as fobj:
             return await self.scan_fileobj(fobj, os.path.basename(to_scan))
 
-    async def search_hash(self, to_scan, hash_type='sha256'):
+    async def search_hash(self, to_scan, hash_type=None):
         """
         Search for a single hash using the PS API asynchronously.
 
         :param to_scan: Hash to search for
-        :param hash_type: Hash type [sha256|sha1|md5]
+        :param hash_type: Hash type [autodetect:sha256|sha1|md5]
```

Comment:
I wasn't clear the first time, this should be `:param hash_type: Hash type [default:autodetect, sha256|sha1|md5]`

Reply:
done